File: Assignment_2/program3.c
//Accept one number from user if number is less than 10 then print "Hello " otherwise print "Demo";


///////////////////////////////////////////////////////////////////////////////
// File Name   :  program3.c
// Description :  Accept one number from user if number is less than 10 then print "Hello " otherwise print "Demo" 
// Autor       :  Ankita Anil Patil
// Date        :  09/05/2025
///////////////////////////////////////////////////////////////////////////////


#include<stdio.h>

void Display(int iNo)
{
    if(iNo < 10)
    {
        printf("Hello\n");
    }
    else
    {
        printf("Demo\n");
    }
}

int main()
{
    int iValue = 0;

    printf("Enter number ");
    scanf("%d",&iValue);

    Display(iValue);

    return 0;
}

File: Assignment_58/program3.java
//Java program to input a string from user and reverse each word of given string  
/*
    Input : ABC XY C
    Output : CBA YX C 
 */

////////////////////////////////////////////////////////////////////////////////
// File Name   :  program3.java
// Description :  Accept string from user and reverse each word of given string 
// Autor       :  Ankita Anil Patil
// Date        :  14/09/2025
///////////////////////////////////////////////////////////////////////////////

import java.util.Scanner; 
class ReverseEachWord  
{       
//ABC XY PQ 
    public  static String  reverseWord(String str) 
    { 
        str = str.trim(); 

        str = str.replaceAll("\\s+"," "); 

        String tokens[] = str.split(" ");
        
        StringBuilder sb = null;

        StringBuilder newstr = new StringBuilder();
        for(String s : tokens)
        {
            sb = new StringBuilder(s);
            newstr.append((sb.reverse()).append(" "));
            
        }

        return new String(newstr);


    } 
}
class program3
{
    public static void main(String[] args)  
    { 
        Scanner sc = new Scanner(System.in); 

        System.out.println("Enput String : "); 
        String str = sc.nextLine(); 

        System.out.println("Input String : "+str); 
        System.out.println("String with Reverese Word : "+ReverseEachWord.reverseWord(str)); 
    } 
}